scheduler/src/main/java/org/pentaho/platform/scheduler2/quartz/QuartzScheduler.java

```diff
@@ -936,4 +943,22 @@ public void fireJobCompleted( IAction actionBean, String actionUser, Map<String,
       }
     }
   }
+
+  public void storeSchedulerStatus() {
+    try {
+      FileUtils.write( getStatusFile(), getStatus().name(), "UTF-8", false );
+    } catch ( IOException | SchedulerException e ) {
+      logger.error( "Error while writing scheduler status to status file", e );
+    }
+  }
+
+  public File getStatusFile() {
+    if ( statusFile == null ) {
+      String schedulerStatusPath =
+        PentahoSystem.getApplicationContext().getSolutionPath( "system" + File.separator + "quartz" ) + File.separator
+          + "scheduler.status";
```

Comment:
Ideally, we'd have a constant to hold the file name, such as in `EmbeddedQuartzSystemListener.DEFAULT_QUARTZ_PROPERTIES_FILE`.